packages/utils/src/nanoid/index.ts

```
@@ -0,0 +1,63 @@
+import * as _nanoid from './nanoid'
+import { isServer } from '../globalConfig'
+
+type TypedArray = Int8Array | Uint8Array | Int16Array | Uint16Array | Int32Array | Uint32Array
+type GetRandomValues = (array: TypedArray) => TypedArray
+type Nanoid = (size?: number) => string
+type CustomAlphabet = (alphabet: string, defaultSize?: number) => Nanoid
+type Random = (p?: void) => number
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)